src/wss/v2/trading_messages.rs

```diff
@@ -227,7 +227,7 @@ pub struct BatchOrderParams {
 pub struct BatchCancelParams {
     pub orders: Vec<IntOrString>,
     pub token: Token,
-    #[serde(rename = "cl_ord_id")]
+    #[serde(rename = "cl_ord_id", skip_serializing_if = "Option::is_none")]
```

Comment:
Did you run into the issue where `#[skip_serializing_none]` has to be above the `#[derive(Serialize)]` to work? It's a weird dependency and compiles (but doesn't skip it).

Reply:
oh I haven't this seems to work. is this an issue that the serde team are aware of? I searched really briefly but didn't find anything

Reply:
Ah, took me a second, but I found it in their closed issues: https://github.com/jonasbb/serde_with/issues/440

Main is now up-to-date with Kraken's changelog, including these changes. I'll be testing locally this week and then I'll release a new major version assuming all goes well.